bundles/core/src/test/java/com/adobe/cq/commerce/core/search/internal/services/SearchResultsServiceImplTest.java

```diff
@@ -429,28 +429,6 @@ public void testSearchWithInvalidSortOrderParam() {
         assertThat(query).contains("sort:{position:ASC}");
     }
 
-    @Test
-    public void testSearchWithInvalidSortKey() {
```

Comment:
So what happens in this case now? I assume this is handled by https://github.com/adobe/aem-core-cif-components/blob/ed3ac61fd2871de50f50f3e346c0726858e6cfbe/bundles/core/src/main/java/com/adobe/cq/commerce/core/search/internal/services/SearchResultsServiceImpl.java#L380-L391 now and allowed. Should this be covered in a separate unit test?

Reply:
Right, we cannot detect the invalid sort field but I log a debug message when the sort field is not known.
I've added unit test for this case too.